src/plugins/plugin.cpp

```diff
@@ -16,11 +17,10 @@ std::string createPluginFilename(std::string_view name, std::string_view extensi
     std::string result;
     result += prefix;
     result += infix;
-    result += toLower(name.data());
-    result += postfix;
-    result += '.';
-    result += extension;
-    return result;
+    result += name.data();
+    result += staticPostfix;
+    result += debugPostfix;
```

Comment:
maybe concat in one line instead of so many += lines? not a big deal anyway

Reply:
I would leave it as is. It would be a long line otherwise.